src/functions-and-arrays.js

```diff
@@ -1,41 +1,130 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(firstNumber, secondNumber) {
+
+  if (firstNumber > secondNumber) {
+    return firstNumber
+  } else {
+    return secondNumber
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arrayOfWords) {
+
+  if (!arrayOfWords || arrayOfWords.length == 0) {
+    console.log("Please pass a non-empty array as function argument.")
+    return null;
+  }
+
+  let longestWord = arrayOfWords[0];
+  for (let i = 1; i < arrayOfWords.length; i++) {
+    if (arrayOfWords[i].length > longestWord.length) {
+      longestWord = arrayOfWords[i];
+    } else {
+      continue;
```

Comment:
`else {continue}` is not neccessary in this case. You have a `for loop `which will last until you reach end of your array -> until `i < arrayOfWords.length;`. And only then you will exit the loop